src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormHeader.tsx

```diff
@@ -51,6 +51,7 @@ export const MyCollectionArtworkFormHeader: React.FC<MyCollectionArtworkFormHead
                       flexDirection="row"
                       justifyContent="space-between"
                       py={2}
+                      minHeight={[70, 90]}
```

Comment:
To make sure the height of the header is the same for each step (even without the skip button).